core-web/package.json

```
@@ -1,9 +1,9 @@
 {
-    "name": "dotcms-ui",
+    "name": "core-web",
     "version": "23.4.0-next.1",
     "license": "MIT",
     "engines": {
-        "node": ">=18.17.0"
+        "node": ">=v18.18.2"
```

I think the standard is to not include the prefix `v`

```suggestion
        "node": ">=18.18.2"
```

---

You are right. Thanks.